style.css

```diff
@@ -320,3 +338,252 @@ nav {
   margin-top: 50px;
   margin-bottom: 10px;
 }
+
+.snap01 {
+  display: none;
+}
+
+.list-1 {
+  list-style: none;
+}
+
+.list-item-1 {
+  display: flex;
+  justify-content: space-between;
+}
+
+.list-2 {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100px;
+  list-style: none;
+}
+
+.list-item-2 {
+  background: #f7f7f9;
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  margin-top: 20px;
+  border-radius: 5px;
+}
+
+.name {
+  margin-left: 10px;
+}
+
+.list-3 {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 30px;
+}
+
+.list-4 {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+@media screen and (min-width: 1024px) {
+  .logo {
+    margin-left: 60px;
+    margin-top: 10px;
+  }
+
+  .mobile-nav {
+    display: none;
+  }
+
+  .desktop-nav {
+    display: inherit;
+    margin-right: 30px;
+  }
+
+  .nav-link {
+    text-decoration: none;
+    margin-right: 30px;
+    color: #344563;
+  }
+
+  .homepage {
+    height: 768px;
+    background-image: url(./images/background_1024.png);
+  }
+
+  .home-page {
+    width: 870px;
+    margin: 100px auto;
+  }
+
+  .intro {
+    width: 350px;
+    height: 122px;
+  }
+
+  /* .description {
+    width: 870px;
+    height: 60px;
+  } */
```

Comment:
- [ ] following your chosen design template I think the color of this `.description` is `#344563;` on desktop version`. Consider uncommenting it and applying the right color so that your desktop design matches that of your template. 

- [ ] Also add `margin-bottom: 12px` to give it space from `.connect` 👍